tests/functional/s3/test_sync_command.py

```diff
@@ -176,7 +176,12 @@ def test_sync_skips_over_files_deleted_between_listing_and_transfer(self):
         # get their stats.
         def side_effect(_):
             os.remove(full_path)
-            raise OSError()
+            # PEP 3151: Python 3.3+ raise a FileNotFoundError, earlier versions
+            # an OSError from os.stat if the file has gone away.
+            try:
+                raise FileNotFoundError(2, "file not found", "foo.txt")
+            except NameError:
+                raise OSError(2, "file not found", "foo.txt")
```

Comment:
`OSError(errno.ENOENT, 'file not found')` will return `FileNotFoundError`, but only if called with the second argument.

Reply:
Oh great, then I'll remove the try/except and adjust the comment (I would not remember this in half a year).